bin/update-tw-manpages

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
I prefer the "Bash strict mode" `set -euo pipefail`, but that could be a matter of taste